docs/manifest.json

```diff
@@ -559,10 +559,28 @@
 	},
 	{
 		"title": "Theme.json Reference",
-		"slug": "theme-json-reference",
```

Comment:
We may need to keep this slug as `theme-json-reference` as there's an existing `theme-json` slug [here](https://developer.wordpress.org/block-editor/how-to-guides/themes/theme-json/).

Reply:
@oandregal This would be `/reference-guides/theme-json`, so I assumed it would be okay. But I forgot to do a search for existing links which I'll take care of now

Reply:
Do you know if there's a way to redirect the old URL to the new one? I don't like that organizing the related docs into a folder will break the existing links

Reply:
> This would be /reference-guides/theme-json, so I assumed it would be okay. But I forgot to do a search for existing links which I'll take care of now

I'm not sure how it works, to be honest.

> Do you know if there's a way to redirect the old URL to the new one? I don't like that organizing the related docs into a folder will break the existing links

Yeah, that's feasible. In the past, we had to do this and it involved creating a track ticket for meta, for example https://meta.trac.wordpress.org/ticket/4388 In general, it's best to avoid asking for redirections too frequently as they are a small team. If we can go with the existing, it's going to be quicker to land this PR.

Reply:
> In general, it's best to avoid asking for redirections too frequently as they are a small team. If we can go with the existing, it's going to be quicker to land this PR.

It seems unavoidable since we want a table of contents sort of page where the old one was now. The redirect that we need is:

`reference-guides/theme-json-reference` -> `reference-guides/theme-json/living`

Reply:
The rest of the names changes seem good to me. I'll get those changes pushed up here soon and ask in #meta on slack to see if they know about using the same slug nested differently

Reply:
Also [asked in #docs](https://wordpress.slack.com/archives/C02RP4WU5/p1642038313140800) and it sounds like we probably [can't reuse a slug](https://wordpress.slack.com/archives/C02RP4WU5/p1642038437141700) which leaves the question: are we okay with https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/ changing to the new ToC page instead of redirecting it to the living reference like it used to?

Reply:
Seeing as we can't reuse slugs, we probably want something more specific than `living`, `v1`, and `migrations`.

Reply:
> are we okay with https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/ changing to the new ToC page instead of redirecting it to the living reference like it was before?

I think this should be fine.

Reply:
I see we use "living" in some places and "unstable" in others (file names, titles, etc). Would you mind consolidating everything to "living"?

Just a personal preference. Now that I see "unstable" it sounds less than ideal, as the theme.json is "stable" in that it works, just it can change. If you feel strongly either way (or any other option), go for it. As long as we use the same name everywhere we're fine.